src/main/java/no/fint/personalmappe/properties/OrganisationProperties.java

```diff
@@ -22,5 +22,6 @@
         private boolean bulk;
         private boolean delta;
         private List<String> personalressurskategori;
+        private List<String> transformationScripts;
```

Comment:
Not sure how entire lists of Javascript functions could be passed through environment variables.  Should probably point to URL instead.

Reply:
Then it could be retrieved from file system (via i.e. ConfigMap), Azure blob or something similar.

Reply:
Well I tested it and it worked - it's just a text string. Think, in the future, we should move it to an Azure blob. Maybe the Azure blob should get the content from a git repo/ci/cd system so that we can add tests also.